packages/@okta/vuepress-site/guides/custom-error-pages/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Customize the Hosted Error Pages
+excerpt: Learn how to customize the Okta hosted error pages to fit your brand or your applicaiton's look and feel.
```

Comment:
>  or your applicaiton's look and feel.

nit: typo for application
Not sure if there is anything on the error page that they can customize to be application specific?

Reply:
Nice catch, thank you. Didn't remove all of the references to application specific content after we talked.